src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: flex;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description {
+    display: flex;
+    flex-direction: column;
+    margin-top: 40px;
+  }
+
+  &__description-title {
+    @extend %size-description-title;
+    color: $main-color;
+  }
+
+  &__description-code {
+    @extend %size-description-code;
+  }
+
+  &__review {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__review-amount {
+    align-self: center;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-cost {
+    @extend %size-price-cost;
+    color: $main-color;
+  }
+
+  &__price-name {
+    @extend %size-price-name;
+    color: $secondary-color;
+  }
+
+  &__button {
+    width: 100%;
+    display: flex;
+    margin-top: 16px;
+    text-align: center;
+    justify-content: center;
+  }
+
+  &__button-link {
+    align-items: center;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 38px;
+    text-transform: uppercase;
+    background-color: $blue-accent-color;
+    border-radius: 5px;
+    padding: 0 69px;
```

Comment:
Please don't use odd values if it is not necessary, just try using full values like `70px`